flow/operations.py

```diff
@@ -104,6 +104,10 @@ class directives(object):
 
     Directives can for example be used to provide information about required resources
     such as the number of processes required for execution of parallelized operations.
+
+    In addition, you can use the `@directives(fork=True)` directive to enforce that a
```

Comment:
```suggestion
    In addition, you can use the `@directives(fork=True)` decorator to enforce that a
```

Reply:
The thing I'm referencing is a decorator. Too confusing?

Reply:
I think the word directive is directly implied by `@directives` but "decorator" might be more helpful for new users. Accept or reject the suggestion at your discretion.

Reply:
I'd prefer to leave as-is.